Documentation/helm-operator.md

```diff
@@ -22,29 +22,29 @@ If role-based access control (RBAC) is enabled in your cluster, you may need to
 
 ```console
 # Create a ServiceAccount for Tiller in the `kube-system` namespace
-kubectl -n kube-system create sa tiller
+kubectl --namespace kube-system create sa tiller
 
 # Create a ClusterRoleBinding for Tiller
 kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
 
 # Patch Tiller's Deployment to use the new ServiceAccount
-kubectl -n kube-system patch deploy/tiller-deploy -p '{"spec": {"template": {"spec": {"serviceAccountName": "tiller"}}}}'
+kubectl --namespace kube-system patch deploy/tiller-deploy -p '{"spec": {"template": {"spec": {"serviceAccountName": "tiller"}}}}'
 ```
 
 ## Installing
 
 The Rook Operator helm chart will install the basic components necessary to create a storage platform for your Kubernetes cluster.
 After the helm chart is installed, you will need to [create a Rook cluster](quickstart.md#create-a-rook-cluster).
 
-The `helm install` command deploys rook on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+The `helm install` command deploys rook on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation. It is recommended that the rook operator be installed into the `rook-system` namespace (you will install your clusters into separate namespaces).
 
 Rook currently publishes builds to the `alpha` and `master` channels. In the future, `beta` and `stable` will also be available.
 
 ### Alpha
 The alpha channel is the most recent release of Rook that is considered ready for testing by the community.
 ```console
 helm repo add rook-alpha https://charts.rook.io/alpha
-helm install rook-alpha/rook
+helm install --namespace rook-alpha/rook
```

Comment:
this one looks to be missing the `rook-system` namespace right after the `--namespace` flag.